src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -122,7 +122,7 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
             setSelectedDayIndex(dayIndex);
           }}
         >
-          {formatDayLabel(day)}
+          <button>{formatDayLabel(day)}</button>
```

Comment:
Why is this a raw `button`? This doesn't seem right. Presumably the `onClick` handler should be on this button, etc.